.ci/build_kubernetes.sh

```diff
@@ -73,12 +75,16 @@ yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spe
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.initContainers[1].volumeMounts[2].mountPath /var/www/html/project-base/app/config/parameters.yml
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.containers[0].volumeMounts[1].mountPath /var/www/html/project-base/app/config/domains_urls.yml
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.containers[0].volumeMounts[2].mountPath /var/www/html/project-base/app/config/parameters.yml
+yq write --inplace project-base/kubernetes/deployments/consumer-patches/product-visibility-recalculate.yml [0].value.volumeMounts[1].mountPath /var/www/html/project-base/app/config/domains_urls.yml
+yq write --inplace project-base/kubernetes/deployments/consumer-patches/product-visibility-recalculate.yml [0].value.volumeMounts[2].mountPath /var/www/html/project-base/app/config/parameters.yml
 
 # Set environment variables to container and initContainer for Google Cloud Storage connection - set it to null
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.containers[0].env[0].value ''
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.containers[0].env[1].value ''
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.initContainers[1].env[0].value ''
 yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.initContainers[1].env[1].value ''
+yq write --inplace project-base/kubernetes/deployments/consumer-patches/product-visibility-recalculate.yml [0].value.env[0].value ''
+yq write --inplace project-base/kubernetes/deployments/consumer-patches/product-visibility-recalculate.yml [0].value.env[1].value ''
 
 # Deploy application using kubectl
```

Comment:
You should wait for the `rabbitmq` rollout when applying the changes (both here and in `.ci/restart_kubernetes.sh`)

Reply:
fixed in cddc6a6a5215af7bb61266eae51c2274a20de426